File: 프로그래머스/unrated/134240. 푸드 파이트 대회/푸드 파이트 대회.py
def solution(food):
    answer = '' 
    for i in range(1,len(food)):
        answer += str(i)*(food[i]//2)
    answer += "0"
    for i in range(1,len(food)):
        answer += str(len(food)-i)*(food[len(food)-i]//2)
    return answer

File: 프로그래머스/unrated/134240. 푸드 파이트 대회/푸드 파이트 대회.js
function solution(food) {
    var answer = "";
    
    for (var i = 1; i <= food.length; i++) {
        answer += String(i).repeat(parseInt(food[i]/2));
    }
    
    answer += "0".repeat(parseInt(food[0]));
    
    for (var i = food.length; i >= 1; i--) {
        answer += String(i).repeat(parseInt(food[i]/2));
    }
    
    return answer;
}